paddle/fluid/inference/api/demo_ci/run.sh

```diff
@@ -17,6 +17,8 @@ if [ $2 == ON ]; then
 fi
 if [ $3 == ON ]; then
   use_gpu_list='true false'
+  DGC_LIB=${inference_install_dir}/third_party/install/dgc/lib
+  export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:${DGC_LIB}
```

Comment:
inference should not depend dgc since it's only for training

Reply:
Done.